test-harness/specs/security-scheme-validation-AC-9.only_oas3.txt

```diff
@@ -0,0 +1,28 @@
+====test====
+Resource is returned when a bearer token is provided as a header.
```

Comment:
Assumed: no validation needed, just token presence is enough for the security to pass.
Maybe the api-key should actually be validated somehow?